### PR TITLE
loadbot: Adjust doge values.

### PR DESCRIPTION
**Pull request by JoeGruffins**
The doge loadbot does not run on master for one because the amount of dcr needed for registration is not sent to the dcr wallet. The lot size makes one lot trades impossible due to it being dust. And for some reason the default feerate for doge is not used so setting explicitly. I would think it should use the client default so can look into why it does not if needed.

Loadbot works on the default dcr_doge after these changes.

### CODE REVIEW

**Comment by JoeGruffins** [comment]
If the rate is much higher than the lot size the bottom graph never loads:
image

Was it wrong to set rate step higher than lot size?

**Comment by JoeGruffins** [comment]
Also noticing the price says btc/dcr rather than doge/dcr

**Comment by buck54321** [comment]
#2669 can help in the future

File: dex/testing/loadbot/mantle.go [hunk 1]
```diff
@@ -734,10 +734,11 @@ func newBotWallet(symbol, node, name string, port string, pass []byte, minFunds,
...
 			Config: map[string]string{
-				"walletname":  name,
-				"rpcuser":     "user",
-				"rpcpassword": "pass",
-				"rpcport":     port,
+				"walletname":   name,
+				"rpcuser":      "user",
+				"rpcpassword":  "pass",
+				"rpcport":      port,
+				"feeratelimit": "40000",
```
Comment (buck54321) [inline dex/testing/loadbot/mantle.go:741]: Do we need this?

Comment (JoeGruffins) [reply dex/testing/loadbot/mantle.go:741]: Apparently. It doesn't use the correct default feeratelimit for doge. It's different for doge correct? Should I look into why? I probably should.

**Comment by JoeGruffins** [comment]
@buck54321 Thanks for that, added.

The bottom graph will not load with rate step higher than lot size. Should I look into that here or a separate issue? I can also swap the market to be doge_dcr I guess. If that is the answer should probably document it. Sorry will look at #2669 have not yet.

**Comment by buck54321** [comment]
> > @buck54321 can rate step be higher than lot size or is there an obvious reason it should not be?
> 
> Seems weird, but I don't think there's any rule about it.

Actually it's not weird at all. Different units even.

**Comment by buck54321** [comment]
> If the rate is much higher than the lot size the bottom graph never loads
> 
> Was it wrong to set rate step higher than lot size?

https://github.com/buck54321/dcrdex/compare/d8e9922...buck54321:dcrdex:formatbiggerrates

**Comment by JoeGruffins** [comment]
> Please add this line to this PR https://github.com/buck54321/dcrdex/compare/d8e9922...buck54321:dcrdex:formatbiggerrates

Oh yeah looks like that solved it. Thanks!

**Review by buck54321 — APPROVED** [review]
Ready to go, but I made some conflicts for you with #2686.

**Comment by JoeGruffins** [comment]
rebased

**Review by buck54321 — APPROVED** [review]

**Pull request merged**